product_docs/docs/pem/9/registering_agent.mdx

```diff
@@ -131,7 +131,7 @@ Postgres Enterprise Manager Agent registered successfully!
 
 Because the `agent_id` of 8 is available, the PEM agent registers successfully. If the given ID is already in use by the existing agent, it throws an error.
 
-Register the PEM agent using the command line. Assign the existing SSL certificates and key files to avoid generating new ones for a particular agent ID in versions 9.0 and later. The SSL certificates and key files must be valid for the database user `agent<ID>` , where `<ID>` must be the same as provided using the command line.  Use the `-o` option.
+Register the PEM agent using the command line. In versions 9.0 and later, assign the existing SSL certificates and key files to avoid generating new ones for a particular agent ID. The SSL certificates and key files must be valid for the database user `agent<ID>`, where `<ID>` must be the same as provided using the command line. Use the `-o` option.
```

Comment:
```suggestion
Register the PEM agent using the command line. Assign the existing SSL certificates and key files to avoid generating new ones for a particular agent ID. The SSL certificates and key files must be valid for the database user `agent<ID>`, where `<ID>` must be the same as provided using the command line. Use the `-o` option.
```
These are version 9 docs, so we can drop this.